src/DGmethods/DGmodel.jl

```diff
@@ -6,126 +6,112 @@ struct DGModel{BL,G,NFND,NFD,GNF}
   gradnumflux::GNF
 end
 
-function (dg::DGModel)(dQdt, Q, param, t; increment=false)
+function (dg::DGModel)(dYdt, Y, param, t; increment=false)
   bl = dg.balancelaw
-  device = typeof(Q.Q) <: Array ? CPU() : CUDA()
+  DFloat = eltype(Y)
+  device = typeof(Y.Q) <: Array ? CPU() : CUDA()
 
   grid = dg.grid
-  topology = grid.topology
-
-  dim = dimensionality(grid)
-  N = polynomialorder(grid)
-  Nq = N + 1
-  Nqk = dim == 2 ? 1 : Nq
-  Nfp = Nq * Nqk
-  nrealelem = length(topology.realelems)
-
-  Qvisc = param.diff
-  auxstate = param.aux
-
-  DFloat = eltype(Q)
-  nviscstate = num_diffusive(bl, DFloat)
+  Ω    = grid.topology.realelems
+  nΩ   = length(Ω)
+  Nᵈ   = dimensionality(grid)
+  N    = polynomialorder(grid)
+  Nq   = N + 1
+  Nqk  = Nᵈ == 2 ? 1 : Nq
+  Nfp  = Nq * Nqk
+  Np   = dofs_per_element(grid)
+
+  σ  = param.diff
```

Comment:
Please retain variables with something like `diff` or `diffusive` and `aux` here, so we do not need a dictionary to translate.

Reply:
I am not sure `aux' is self-explanatory. You will always end up needing a dictionary. I think the issue here is to find the right balance between using somewhat self-explanatory variable names without ending up with equations that are too long and verbose. There is a reason why we introduce symbols in math :-)